<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuctionPage</class>
 <widget class="QDialog" name="AuctionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuctionPage</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>0</y>
     <width>16</width>
     <height>601</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_username">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>余额：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_money">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>40 元</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>拍卖中：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>190</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>该房间的其他用户：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_leaveAuction">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>离开房间</string>
   </property>
  </widget>
  <widget class="Line" name="lin">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>170</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_itemName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>拍卖商品</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lwg_other">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>220</y>
     <width>141</width>
     <height>161</height>
    </rect>
   </property>
   <property name="selectionRectVisible">
    <bool>true</bool>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>451</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请出价：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_startChat_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>365</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出价</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前价格：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_logout_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>25</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>商品详情</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_curPrice">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>50 元</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_timeRemain">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>30 S</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>剩余时间：</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
